include/dp_vnf.h

```diff
@@ -42,6 +42,17 @@ struct dp_vnf {
 	struct dp_vnf_prefix	alias_pfx;
 };
 
+struct dp_vnf_value_key {
```

Comment:
@byteocean 
If this is supposed to be reverse lookup of the original hashtable, why do we  redefine a dp_vnf as dp_vnf_value_key ?
dp_vnf can be used as key for reverse lookup ? I need some more background why this needs to be this way.

Reply:
Why cant we just put a packed to the original structure ?

Reply:
yeah, we need to use a data structure without any padding bits to serve as the table key. Although we can make `struct dp_vnf` packed, but we cannot make its nested member `struct dp_ip_address` packed.

Reply:
Ok. Just talked to Jay and he also explained me why.